system-variables.md

```diff
@@ -1112,6 +1112,24 @@ MPP is a distributed computing framework provided by the TiFlash engine, which a
 - Default value: `OFF`
 - This variable is used to set whether the `AUTO_INCREMENT` property of a column is allowed to be removed by executing `ALTER TABLE MODIFY` or `ALTER TABLE CHANGE` statements. It is not allowed by default.
 
+### tidb_analyze_column_options <span class="version-mark">New in v8.3.0</span>
+
+> **Note:**
+>
+> This variable only works when [`tidb_analyze_version`](#tidb_analyze_version-new-in-v510) is set to `2`.
+>
+> After upgrading a TiDB cluster from a version earlier than v8.3.0, this variable be set to `ALL` to keep the original behavior.
+>
+> Since v8.3.0, for a newly deployed TiDB cluster, this variable is set to `PREDICATE` by default.
```

Comment:
The word "since" is usually used with the present perfect tense.
```suggestion
> Starting from v8.3.0, for a newly deployed TiDB cluster, this variable is set to `PREDICATE` by default.
```